types/types.ts

```diff
@@ -53,127 +29,86 @@ interface FeatureFlags {
   port?: number;
 }
 
-interface MetricsResults {
-  timestamps: Timestamp[];
+export interface MetricsResults {
   timings: Timing[];
   generatedTime: string;
   lighthouseVersion: string;
-  url: string;
-  initialUrl: string;
+  requestedUrl: string;
+  finalUrl: string;
 }
 
-interface PWMetricsResults {
+export interface PWMetricsResults {
   runs: MetricsResults[];
   median?: MetricsResults;
 }
 
-interface MetricsDefinition {
-  name: string;
-  id: string;
-  getTs(audits: LighthouseAudits): number;
-}
-
-interface Timestamp {
-  title: string;
-  id: string;
-  timestamp: number;
-}
-
-interface Timing {
+export interface Timing {
   title: string,
   id: string;
-  timestamp: number;
   timing: number;
   color: string;
 }
 
-interface AuthorizeCredentials {
+export interface AuthorizeCredentials {
   installed: {
     client_secret: number;
     client_id: number;
     redirect_uris: Array<any>;
   }
 }
 
-interface Oauth2Client {
+export interface Oauth2Client {
   generateAuthUrl: Function;
   getToken: Function;
   credentials: any;
 }
 
-interface GSheetsValuesToAppend {
+export interface GSheetsValuesToAppend {
   0: string; // lighthouseVersion
   1: string; // url
   2: string; // time
-  3: number; // TTFCP timing
-  4: number; // TTFMP timing
-  5: number; // PSI timing
-  6: number; // FV timing
-  7: number; // VC100 timing
-  8: number; // TTI timing
-  9: number; // VC85 timing
+  3: number; // TTFMP timing
+  4: number; // PSI timing
+  5: number; // VISUALLY_COMPLETE timing
+  6: number; // TTFI timing
+  7: number; // TTCI timing
```

Comment:
since these are changing position, i guess these sheets will be a bit broken now.

Reply:
nice catch 👍